index.html

```diff
@@ -290,6 +291,33 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
+  <section class="contact-card">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+
+      <p>
+        If you have an application you are interested in developing, a
+        features that you need built or a project that need coding. I'd love
+        to help with it.
+      </p>
+    </div>
+    <div class="contact-me-form">
+      <form class="form" action="https://formspree.io/f/xbjeoaeo" method="Post" id="form">
+        <label hidden for="name">Name</label>
+        <input type="text" name="name" class="input name-input" id="username" placeholder="name" maxlength="30"
+          required />
+        <label hidden id="mylabel" for="email">Email</label>
+        <input type="email" name="email" class="input mail-input" id="email" placeholder="enter email" size="64"
+          maxlength="64" required />
+        <label hidden id="message" for="message">Message</label>
+        <textarea name="message" class="input-textarea text-area" id="msg" cols="30" maxlength="500"
+          placeholder="messages" required>
+        </textarea>
```

Comment:
- [ ] Hi Team, you have don really well, I found one small bug that we can fix and then the project is completed, so currently there is some space or tab in your textarea element bydefault you can not even see the placeholder messages and this makes it possible to send empty messages in email :point_down: 
![image](https://user-images.githubusercontent.com/97521557/220538315-740cf87e-b52b-4e74-aaef-054ab64e234f.png)

I think you can fix this by making the text area like below rather than closing in a newline close it just after :point_down: 
`<textarea name="message" class="input-textarea text-area" id="msg" cols="30" maxlength="500" placeholder="messages" required></textarea>`